services/storage/src/simcore_service_storage/modules/celery/tasks.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import time
+
+
+from celery import Task
+from common_library.errors_classes import OsparcErrorMixin
+from models_library.progress_bar import ProgressReport
+from models_library.projects_nodes_io import StorageFileID
+from servicelib.logging_utils import log_context
+
+from .utils import get_celery_worker
+
+_logger = logging.getLogger(__name__)
+
+
+def export_data(task: Task, files: list[StorageFileID]):
```

Comment:
Q: I gues this is where I need to put my code.

Reply:
Yes, yes